app/controllers/items_controller.rb

```diff
@@ -169,25 +162,24 @@ def release_hold
       return
     end
     Dor::WorkflowService.update_workflow_status 'dor', @object.pid, 'accessionWF', 'sdr-ingest-transfer', 'waiting'
+    if params[:bulk]
+      render :status => 200, :text => 'Updated!'
+      return
+    end
     respond_to do |format|
-      if params[:bulk]
-        render :status => 200, :text => 'Updated!'
-        return
-      end
       format.any { redirect_to catalog_path(@object.pid), :notice => 'Workflow was successfully updated' }
     end
   end
 
   def embargo_update
-    if current_user.is_admin
-      new_date = DateTime.parse(params[:embargo_date])
-      @object.update_embargo(new_date)
-      @object.datastreams['events'].add_event('Embargo', current_user.to_s , 'Embargo date modified')
-      respond_to do |format|
-        format.any { redirect_to catalog_path(params[:id]), :notice => 'Embargo was successfully updated' }
-      end
-    else
+    unless current_user.is_admin
       render :status => :forbidden, :text => 'forbidden'
+      return
+    end
+    @object.update_embargo(DateTime.parse(params[:embargo_date]))
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.